src/components/Notifications.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+    <v-bottom-sheet :model-value="hasAnyPushedNotificationsThatShouldBeVisible">
+        <v-card>
+            <v-card-item>
+                <v-card-title>Notifications</v-card-title>
+            </v-card-item>
+            <v-card-text>
+                <div class="h-auto d-flex flex-column align-stretch ga-2">
+                    <v-alert
+                        v-for="notification in pushedNotifications"
```

Comment:
maybe you want to limit how many can be visible at once
speaking of experience...

Reply:
![image](https://github.com/MiSArch/frontend/assets/53957498/1dbbda29-3d90-4b2a-b478-30f08b0901dc)
This is what I mean XD
Just to be clear, this was my fault, as I was not logged in.
But it can show what can happen if sth goes wrong.
Also, I do not really like that error messages are in a blocking dialog.
Just imagine sth goes repeatedly wrong, then the dialog would open every short interval, and would prevent you from doing anything with the frontend.
The same goes for notifications: you do not want to get a blocking dialog the second you receive a notification (e.g. your stuff has been delivered), as it interrupts you from what you are currently doing and thus enforces a (bad) context switch

Reply:
How about this: For now, I fix the problem regarding the lack of limitation and we create an Issue where the "Notification Center" must be changed into something that doesn't interrupt the user in their workflow. Because I think there are other issues that have a higher priority.
What do you think?